.github/CODEOWNERS

```diff
@@ -1 +1,2 @@
-* @equinixmetal-helm/maintainers
\ No newline at end of file
+* @equinixmetal-helm/maintainers
+* @metal-toolbox/provisioning-core
```

Comment:
I think this will need to be a team defined in the `equinixmetal-helm` org here to work?

Reply:
Oh, I guess I didn't realize what the format of the file was. I've updated it to list both groups on the same line, which should work?

Reply:
I've now removed the CODEOWNERS change and will look to get the team added to the helm maintainers group instead.